core/services/relay/evm/functions/offchain_config_digester.go

```diff
@@ -0,0 +1,60 @@
+package functions
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/chains/evmutil"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+var (
+	_                     types.OffchainConfigDigester = FunctionsOffchainConfigDigester{}
+	FunctionsDigestPrefix                              = types.ConfigDigestPrefixEVM
+	// In order to support multiple OCR plugins with a single jobspec & OCR2Base contract, each plugin must have a unique config digest.
+	// This is accomplished by overriding the single config digest from the contract with a unique prefix for each plugin via this custom offchain digester & config poller.
+	ThresholdDigestPrefix = types.ConfigDigestPrefix(7)
+	S4DigestPrefix        = types.ConfigDigestPrefix(8)
+)
+
+type FunctionsOffchainConfigDigester struct {
+	PluginType   FunctionsPluginType
+	BaseDigester evmutil.EVMOffchainConfigDigester
```

Comment:
My suggestion was to embed (https://gobyexample.com/struct-embedding) but that's OK too.